src/util/types.ts

```diff
@@ -29,61 +30,61 @@ export function parseType(type: AnySSZType): FullSSZType {
   if (isFullSSZType(type)) {
     return type as FullSSZType;
   }
-  if(typeof type === "string") {
-    if (type === "bool") {
+  if(typeof type === 'string') {
```

Comment:
parseType is growing significantly, have you consider extraction expansion of each type to separate function like: `byteVectorToFullSSZType`, `arrayToFullSSSZType` ?

Reply:
Yeah, I can do that in the next PR. We'll be adding two more base types (bitvector and bitlist).